Example/ExampleAPI/Endpoint/Base/UploadEndpoint+Extension.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  UploadEndpoint.swift
+//  ExampleAPI
+//
+//  Created by Aleksei Tiurnin on 15.01.2021.
+//  Copyright © 2021 RedMadRobot. All rights reserved.
+//
+
+import Apexy
+
+extension UploadEndpoint {
+
+    public typealias ErrorType = Error
+    
+    public func error(from response: URLResponse?, with body: Data?, and error: Error) -> Error {
```

Comment:
IMHO using parameters' names like `with`, `and` in this method can lead to misunderstanding. 
```swift
error(from response: URLResponse?, withBody body: Data?, withError error: Error) -> Error
```
or 
```swift
error(fromResponse response: URLResponse?, withBody body: Data?, withError error: Error) -> Error
``` 
is much better.

Reply:
Agree. This naming can be misleading. Fixed it👍